app/components/App.js

```diff
@@ -15,18 +15,18 @@ export default class App extends Component {
 
   }
 
```

Comment:
I think we should add a comment saying what this is.
//Anonymous user sign in, could be used in the future